driver-core/src/main/com/mongodb/internal/connection/tlschannel/async/AsynchronousTlsChannelGroup.java

```diff
@@ -702,7 +706,7 @@ public void shutdownNow() {
      *
      * @return whether the channel is terminated
      */
-    public boolean isTerminated() {
+    private boolean isTerminated() {
```

Comment:
This change seems irrelevant to making sure that the rest of the `tlschannel` code can't terminate a user-supplied executor. If that's so, let's not do the change.